receiptui/build.gradle

```diff
@@ -3,7 +3,6 @@ project.ext {
     fileFilter = ['**/BuildConfig.*',
                   '**/com/hyperwallet/android/ui/receipt/HyperwalletReceiptUi.*',
                   '**/com/hyperwallet/android/ui/receipt/view/*.*',
-                  '**/com/hyperwallet/android/ui/receipt/viewmodel/*.*'
```

Comment:
Why is the build failing?

Reply:
@fmattos-hw  the commend given by @skoong is getting failed - "/gradlew clean jacocoTestCoverageVerification" check the attached screenshot, look like it gets failed due to not meet the test coverage 
<img width="1617" alt="command test report" src="https://user-images.githubusercontent.com/52742232/63496409-80d98180-c4df-11e9-8778-7fbeadc4d580.png">